WinRT.Runtime/Projections/IDictionary.net5.cs

```diff
@@ -14,12 +14,13 @@
 
 namespace Windows.Foundation.Collections
 {
+    //Need to rethink how to name/define this interface
     [Guid("3C2925FE-8519-45C1-AA79-197B6718C1C1")]
     interface IMap<K, V> : IIterable<IKeyValuePair<K, V>>
     {
         V Lookup(K key);
         bool HasKey(K key);
-        IMapView<K, V> GetView();
+        IReadOnlyDictionary<K, V> GetView(); 
```

Comment:
comment on this line specifically - that combining IMap & IReadOnlyDictionary needs redesign